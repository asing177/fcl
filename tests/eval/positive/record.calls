go(C2(0, false))
