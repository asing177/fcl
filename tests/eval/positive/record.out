*** Deltas ***
[ ModifyGlobal Name { unName = "x" } (VNum (NumDecimal 0))
, ModifyState WorkflowState { places = fromList [ PlaceEnd ] }
, Terminate
]

*** Globals ***
fromList
  [ ( Key { unKey = "testAddr" }
    , VAccount (Address "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve")
    )
  , ( Key { unKey = "testAddr2" }
    , VAccount (Address "Fw68hsis6JPW4ed3ER1VC959TTf9DqberLxrq5uECKxh")
    )
  , ( Key { unKey = "testAddr3" }
    , VAccount (Address "4Stv5gWGt78D3p919HFcKEschSmyGKGrAX2G35xJgdhw")
    )
  , ( Key { unKey = "x" } , VNum (NumDecimal 0) )
  ]
